skll/data/featureset.py

```diff
@@ -469,30 +546,34 @@ def __getitem__(self, value):
             )
         else:
             label = self.labels[value] if self.labels is not None else None
-            feats = self.features[value, :]
-            features = (
-                self.vectorizer.inverse_transform(feats)[0] if self.features is not None else {}
-            )
+            feats = self.features[value, :] if self.features is not None else None
+            if self.vectorizer:
+                features = (
+                    self.vectorizer.inverse_transform(feats)[0] if self.features is not None else {}
```

Comment:
what will the return value be if feats is None?

Reply:
Good catch! Probably `feats` should be an empty numpy array for it go through?